src/sentry/backup/imports.py

```diff
@@ -28,79 +29,36 @@ class OldImportConfig(NamedTuple):
     use_natural_foreign_keys: bool = False
 
 
-def imports(src, old_config: OldImportConfig, printer=click.echo):
-    """Imports core data for the Sentry installation."""
+def _import(src, scope: ImportScope, old_config: OldImportConfig, printer=click.echo):
```

Comment:
You could use `__all__` in the module to define an allow list of symbols that can be imported. This will let you hide `_import` and require folks to use the public interfaces.

Reply:
Done.